invenio/b2share/modules/main/templates/footer.html

```diff
@@ -34,6 +34,7 @@
                 <a href="{{ request.url_root+'docs/b2share-tou' }}">Terms of Use</a>
                 <a href="{{ request.url_root+'docs/b2share-faq' }}">FAQ</a>
                 <a href="{{ request.url_root+'docs/b2share-guide' }}">User&nbsp;Guide</a>
+                <a href="{{ request.url_root+'docs/b2share-rest-api' }}">REST&nbsp;API</a>
```

Comment:
I feel that we have too many links in the bottom of the page, maybe we should talk to Hilary (or our designer) and decide to put them somewhere else.